docs/csharp/quick-starts/introduction-to-classes.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Quick Starts - Introduction to Classes - C# Guide
+description: Create your first C# program and explore object oriented concepts
+keywords: C#, Get Started, Lessons, Classes, Object Oriented Programming
+author: billwagner
+ms.author: wiwagn
+ms.date: 10/11/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+---
+# Introduction to classes
+
+This lesson assumes that you've installed [.NET Core](http://dot.net/core) and [Visual Studio Code](https://code.visualstudio.com/), or [Visual Studio](https://www.visualstudio.com/) on Mac or Windows.
```

Comment:
I didn't see anything specific to VS Code or Visual Studio in this tutorial. So not sure if I'd list it as a requirement. Perhaps you can suggest they make the edits using a code editor like VS Code or Visual Studio?